radio/src/targets/horus/telemetry_driver.cpp

```diff
@@ -20,9 +20,12 @@
 
 #include "opentx.h"
 
-DMAFifo<512> telemetryFifo __DMA (TELEMETRY_DMA_Stream_RX);
+DMAFifo<TELEMETRY_FIFO_SIZE> telemetryDMAFifo __DMA (TELEMETRY_DMA_Stream_RX);

```

Comment:
these 2 buffers could be merged to save around 512 bytes RAM , but another class would be needed